here-naksha-lib-base/src/commonMain/kotlin/naksha/base/Platform.kt

```diff
@@ -115,6 +115,8 @@ expect class Platform {
          */
         val logger: PlatformLogger
 
+        fun deepEquals(obj1: Any?, obj2: Any?): Boolean
```

Comment:
Why is this a platform specific implementation, it seems it would perfectly work as cross-platform code, so added into `PlatformUtil` as `deepEquals`?

Reply:
will do, I was just not familiar enough with the multiplatform layout